marklogic/datadog_checks/marklogic/api.py

```diff
@@ -0,0 +1,99 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import Any, Dict
+
+from datadog_checks.base.utils.http import RequestsWrapper
+
+
+class MarkLogicApi(object):
+    def __init__(self, http, api_url):
+        # type: (RequestsWrapper, str) -> None
+        self._http = http
+        self._base_url = api_url + '/manage/v2'
```

Comment:
`/manage/v2` is used here and in `resources.py`, I think it should be a constant, to be sure they match.